tests/core/block/e2e/test_correctness.py

```diff
@@ -316,6 +297,11 @@ def test_v1_v2_greedy_equality_prefix_caching_enabled_with_preemption(
 
     NOTE: We want a significant number of generated tokens so that any incorrect
     KV mapping has time to build up error.
+
+    NOTE(Kuntai): Though we have removed block manager v1, this test is still
+    useful as it asserts the behavior of block manager v2 (now it is called 
+    SelfAttnBlockManager) is the same when swapping / preemption, so we keep 
+    this test.
```

Comment:
Ditto. We should rename the test to reflect what's actually being tested.

Reply:
Oh yes ur right. Fixed.